Packs/AutofocusFeed/Integrations/AutofocusFeed/AutofocusFeed.py

```diff
@@ -0,0 +1,242 @@
+
+    Returns:
+        'ok' if test passed, anything else will fail the test.
+    """
+    indicator_feeds = client.indicator_feeds
+    if 'Daily Threat Feed' in indicator_feeds:
+        client.indicator_feeds = ['Daily Threat Feed']
+        try:
+            client.build_iterator()
+        except Exception:
+            raise Exception("Could not fetch Daily Threat Feed\n\nCheck your API key and your connection AutoFocus.")
+
+    if 'URL Feed' in indicator_feeds:
+        client.indicator_feeds = ['URL Feed']
+        try:
+            client.build_iterator()
+        except Exception:
+            raise Exception(f"Could not fetch URL Feed {client.url_feed_suffix}\n\n"
+                            f"Check your API key the URL Feed ID and Name and Check if they are Enabled in AutoFocus.")
+    return 'ok', {}, {}
+
+
+def get_indicators_command(client: Client, args: dict):
+    """Initiate a single fetch-indicators
+
+    Args:
+        client(Client): The AutoFocus Client.
+        args(dict): Command arguments.
+
+    Returns:
+        str, dict, list. the markdown table, context JSON and list of indicators
+    """
+    indicators = client.build_iterator()
+    if args.get('offset'):
+        indicators = indicators[int(str(args.get('offset'))):]
+
+    if args.get('limit'):
+        indicators = indicators[:int(str(args.get('limit')))]
+
+    hr_indicators = []
+    for indicator in indicators:
+        hr_indicators.append({
+            "Value": indicator.get('value'),
+            "Type": indicator.get('type')
+        })
+
+    human_readable = tableToMarkdown("Indicators from AutoFocus:", hr_indicators,
+                                     headers=['Value', 'Type'], removeNull=True)
+
+    if args.get('limit'):
+        human_readable = human_readable + f"\nTo bring the next batch of indicators run:\n!get-indicators " \
+            f"limit={args.get('limit')} offset={int(str(args.get('limit'))) + int(str(args.get('offset')))}"
+
+    return human_readable, {
+        f"{SOURCE_NAME}.Indicator(val.Value === obj.Value && val.Type === obj.Type)": hr_indicators
+    }, indicators
+
+
+def fetch_indicators_command(client: Client):
+    """Fetch-indicators command from AutoFocus Feeds
+
+    Args:
+        client(Client): AutoFocus Feed client.
+
+    Returns:
+        list. List of indicators.
+    """
+    indicators = client.build_iterator()
+
+    return indicators
+
+
+def main():
+    params = demisto.params()
+
+    client = Client(params.get('api_key'),
+                    params.get('insecure'),
+                    params.get('proxy'),
+                    params.get('indicator_feeds'),
+                    params.get('output_feed_id'),
+                    params.get('output_feed_name'))
+
+    command = demisto.command()
+    demisto.info(f'Command being called is {command}')
+    # Switch case
+    commands = {
+        'test-module': module_test_command,
+        'get-indicators': get_indicators_command
```

Comment:
Add the integration name or some shortened version of the integration's name as a prefix to the command. Need to update here and in your yaml

Reply:
done